### ISSUE TITLE
Union argument leads to error

### ISSUE THREAD

**Issue by nerdoc** [adamghill/django-unicorn#392]
When having a Union argument type, or Python's new-style 'or' argument type "|", Unicorn crashes. Given the following code:
```python
class MyComponentView(UnicornView):
    state: Union[bool,None] = None

    # or
    # def my_method(state:  bool | None) -> None
    def my_method(state: Union[bool, None]) -> None:  
        self.state=state
```
When you cal my_method from the client, Unicorn crashes:
```
  File "/.../.venv/lib/python3.10/site-packages/django_unicorn/views/action_parsers/call_method.py", line 118, in _call_method_name
    if issubclass(type_hints[argument], Model):
TypeError: issubclass() arg 1 must be a class
```

This is because `argument` is UnionType here, says the debugger: 
![image](https://user-images.githubusercontent.com/2955584/160710624-3906a1fc-fadb-418b-8d6f-0efc2453cdab.png)

So I suppose this could be done better...?

**Comment by nerdoc** [comment]
This also fails when using `typing.Optional[bool]`. Same error.

**Comment by adamghill** [comment]
Oh, I think I need to handle the type annotations a little smarter than I currently do. I'll take a look at this and see how to fix it.

**Comment by nerdoc** [comment]
Seems to be a Python problem which is better handled in py38 - if you want to support py36 onwards, have a look at https://stackoverflow.com/a/45959000/818131

**Comment by adamghill** [comment]
Thanks for that SO answer! That provides the clues that are needed to make this type annotation viable.

**Comment by adamghill** [comment]
Since `Unicorn` doesn't _really_ use the type annotations here (unlike, say Pydantic) I just skip over this error if it happens. I think that should fix your issue, but let me know if it doesn't in the next release.

**Comment by adamghill** [comment]
Should be fixed in 0.44.1 which I just deployed: https://pypi.org/project/django-unicorn/0.44.1/.
